website/docs/d/image.html.markdown

```diff
@@ -26,11 +26,14 @@ output "image_id" {
 
 ## Argument Reference
 
-* `name` - (Required) The name of the Image.
+* `name` - (Optional) The name of the Image.
+* `name_regex` - (Optional) Regex pattern of the image to match.
+* `name_descending` - (Optional) By default when matching by regex, images are sorted by name in ascending order and the first match is chosen, to sort descending, set this flag.
```

Comment:
would it be instead worth making this a `most_recent` flag as in the AWS data source?